docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -541,6 +541,39 @@ You may have trouble in the configuration if you see Pod statuses like `RunConta
    [`Vagrantfile`][ubuntu-vagrantfile] for how this can be achieved.
 
 [ubuntu-vagrantfile]: https://github.com/errordeveloper/k8s-playground/blob/22dd39dfc06111235620e6c4404a96ae146f26fd/Vagrantfile#L11)
+2. If you were using CentOS, while you get stuck while setup master node. you can follow this guide 
+
+```
```

Comment:
Follow the style guide for [code blocks](https://kubernetes.io/docs/home/contribute/style-guide/#code-snippet-formatting). Make sure that code blocks contain code, not procedural information.